inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml

```diff
@@ -0,0 +1,122 @@
+                    { 0.0, 0.0, 0.1 } }"
+      autoSpaceRadialElems="{ 0, 0, 1 }"
+      useCartesianOuterBoundary="3"
+      cellBlockNames="{ casing, cement, rock }"
+      />
+  </Mesh>
+  <!-- SPHINX_WellboreMeshEnd -->
+
+  <Events
+    maxTime="1e5">
+
+    <PeriodicEvent
+      name="stressHistoryCollection_casing"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/stressCollection_casing"/>
+    <PeriodicEvent
+      name="stressHistoryCollection_cement"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/stressCollection_cement"/>
+    <PeriodicEvent
+      name="stressHistoryCollection_rock"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/stressCollection_rock"/>
+    <PeriodicEvent
+      name="stressTimeHistoryOutput_casing"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/stressHistoryOutput_casing"/>
+    <PeriodicEvent
+      name="stressTimeHistoryOutput_cement"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/stressHistoryOutput_cement"/>
+    <PeriodicEvent
+      name="stressTimeHistoryOutput_rock"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/stressHistoryOutput_rock"/>
+
+    <PeriodicEvent
+      name="temperatureHistoryCollection_casing"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/temperatureCollection_casing"/>
+    <PeriodicEvent
+      name="temperatureHistoryCollection_cement"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/temperatureCollection_cement"/>
+    <PeriodicEvent
+      name="temperatureHistoryCollection_rock"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/temperatureCollection_rock"/>
+    <PeriodicEvent
+      name="temperatureTimeHistoryOutput_casing"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/temperatureHistoryOutput_casing"/>
+    <PeriodicEvent
+      name="temperatureTimeHistoryOutput_cement"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/temperatureHistoryOutput_cement"/>
+    <PeriodicEvent
+      name="temperatureTimeHistoryOutput_rock"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/temperatureHistoryOutput_rock"/>
+
+    <!-- SPHINX_DisplacementPeriodicEvent -->
+    <PeriodicEvent
+      name="displacementHistoryCollection"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Tasks/displacementCollection"/>
+    <PeriodicEvent
+      name="displacementTimeHistoryOutput"
+      endTime="1e5"
+      forceDt="1e4"
+      target="/Outputs/displacementHistoryOutput"/>
+    <!-- SPHINX_DisplacementPeriodicEventEnd -->
+
+    <PeriodicEvent
+      name="outputs"
+      timeFrequency="1e4"
+      target="/Outputs/vtkOutput"/>
+
+    <PeriodicEvent
+      name="solverApplications"
+      maxEventDt="1e3"
+      target="/Solvers/thermoElasticSolver"/>
+
+    <!--PeriodicEvent
+      name="restarts"
+      timeFrequency="1e5"
+      target="/Outputs/restartOutput"/-->
```

Comment:
You can keep the restart and trigger at the middle of the simulation (5e4) so that your xml is ready for the rebaseline

Reply:
Done